http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala

```diff
@@ -842,6 +850,13 @@ object RuntimeServiceInterp {
         else req.scopes //default to create gce runtime if runtimeConfig is not specified
     }
 
+    // Do not enable welder for rstudio
+    val welderEnabled =
```

Comment:
Could this just be:
```
val welderEnabled = clusterImages.map(_.imageType).contains(Welder)
```
?

Since we already have the logic to exclude welder for RStudio runtimes in `getRuntimeImages`, we shouldn't need to duplicate it here.